packages/common/src/authorization/projectMemberAbility.test.ts

```diff
@@ -1371,19 +1371,22 @@ describe('Project member permissions', () => {
                 ability.can(
                     'create',
                     subject('Project', {
-                        projectUuid,
+                        upstreamProjectUuid: projectUuid,
```

Comment:
do we want to create DEFAULT projects with upstreamProjectUuid ?

Reply:
in fact, we throw an error here https://github.com/lightdash/lightdash/pull/12158/files#diff-dba02b88a0bd4a4fdcf7ee49daaaa481189bc9ad76a683c357d0f59e568370a5R299

I guess this test is not very relevant then.

Reply:
Yeah, I think it's still good to keep it, since this only tests the project member's ability without needing to know what is happening within services or organization permissions.

btw, this check returns false so that permission check also forbids the action